### PR TITLE
Documentation restructuring and updates

### PR DESCRIPTION
**Pull request by shelane**
- Restructured the documentation for the Algolia size threshold
- Updated ssh agent information with the new ssh-key command replacing ssh-add
- Fixed typo with virtual host listing
- Added logic for credits page to show "1 commit" instead of "1 commits"

### CODE REVIEW

**Comment by shelane** [comment]
I have moved the Docksal image section under "Default stacks" in the Understanding Stack Configuration page and moved the Automate the initialization process to Getting Started/Project Setup. Reading through makes sense to me, but it's your call.

**Comment by shelane** [comment]
It appears that the branch on netlify didn't rebuild after the pushed changes.

File: docs/content/getting-started/project-setup.md [hunk 1]
```diff
@@ -64,7 +64,8 @@ which would require an ssh key for access).
 
 Your project stack is now running. Access it in your browser: `http://myproject.docksal`
 
-!!! tip "VIRTUAL HOST name"
-    By default the virtual host name is equal to the project's folder name sans spaces and dashes, 
-    with the `.docksal` domain appended to it.  
-    `myproject => myproject.docksal`
+{{% notice tip %}} "VIRTUAL HOST name"
+By default the virtual host name is equal to the project's folder name sans spaces and dashes,
```
Comment (achekulaev) [inline docs/content/getting-started/project-setup.md]: @shelane dashes are allowed

Comment (shelane) [reply docs/content/getting-started/project-setup.md]: That’s a relic from before. I didn’t even read it. I’ll update the text.

**Pull request merged**